hibernate-core/src/main/java/org/hibernate/dialect/OracleDialect.java

```diff
@@ -643,6 +653,79 @@ public void contributeTypes(TypeContributions typeContributions, ServiceRegistry
 
 			typeContributions.contributeSqlTypeDescriptor( descriptor );
 		}
+		typeContributions.contributeSqlTypeDescriptor( new OracleBooleanTypeDescriptor() );
+	}
+
+	private static final class OracleBooleanTypeDescriptor extends BooleanTypeDescriptor {
+
+		private static final Logger log = CoreLogging.logger( BasicBinder.class );
+
```

Comment:
We should probably instead create a "subsystem" logger here and use that from the various impls.  More just a mental todo...